packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache

```diff
@@ -1,7 +1,7 @@
 <div id="main-content" class="card pair-auth">
   <header>
     <h1 id="fxa-pair-auth-allow-header">
-      {{#unsafeTranslate}}Approve connections <small>to begin syncing</small>{{/unsafeTranslate}}
+      {{#unsafeTranslate}}Approve new device <small>for %(email)s</small>{{/unsafeTranslate}}
```

Comment:
I wasn't able to test this physically but the mockups have email address twice in the screen. Once here and again below the location. Is this intended?

![Screen Shot 2019-04-23 at 8 54 04 AM](https://user-images.githubusercontent.com/1295288/56582492-629d5c80-65a5-11e9-88c1-4cee2b8a7612.png)

Reply:
@vladikoff will this occur on other screens too?

Reply:
@ryanfeeley do you want the text duplicated?

Reply:
Oops! My bad guys! I removed it.
<img width="348" alt="image" src="https://user-images.githubusercontent.com/68213/56602006-845d0a80-65ca-11e9-9300-12d9db472438.png">

Reply:
Thanks @ryanfeeley  !